pkg/dump/dump.go

```diff
@@ -123,6 +125,7 @@ func ReplyMessage(kind, name string, rpl interface{}, latency time.Duration, syn
 			dir:     reply,
 			kind:    kind,
 			method:  name,
+			qualif:  qualif,
```

Comment:
super nit: "qualif" just looks very odd abbreviation, "qualifier" would not be that much longer and would be more descriptive

Reply:
Updated, changed to `qualifier`.